skbio/diversity/__init__.py

```diff
@@ -377,15 +377,15 @@
    >>> plt.close('all') # not necessary for normal use
    >>> fig = sample_md.boxplot(column='Faith PD', by='body_site')
 
-We can also compute Spearman correlations between all pairs of columns in this
-``DataFrame``. Since our alpha diversity metrics are the only two numeric
-columns (and thus the only columns for which Spearman correlation is relevant),
-this will give us a symmetric 2x2 correlation matrix.
+    We can also compute Spearman correlations between all pairs of columns in
```

Comment:
The reason I indented them was that they weren't indented and stood out w.r.t. the rest of the example text (see [here](https://github.com/biocore/scikit-bio/blob/master/skbio/diversity/__init__.py#L380))